crypto/dilithium/pqcrystals_dilithium_ref_common/polyvec.c

```diff
@@ -40,33 +42,56 @@ void polyvec_matrix_pointwise_montgomery(ml_dsa_params *params,
                                          const polyvecl *mat,
                                          const polyvecl *v) {
   unsigned int i;
-
-  for(i = 0; i < params->k; ++i)
+  for(i = 0; i < params->k; ++i) {
     polyvecl_pointwise_acc_montgomery(params, &t->vec[i], &mat[i], v);
+  }
 }
 
 /**************************************************************/
 /************ Vectors of polynomials of length L **************/
 /**************************************************************/
 
+/*************************************************
+* Name:        polyvecl_uniform_eta
+*
+* Description: FIPS 204: Algorithm 33 ExpandS (for vectors l).
+*              Samples vector v with polynomial coordinates whose
+*              coefficients are in [-eta, eta].
+*
+* Arguments:   - ml_dsa_params: parameter struct
+*              - polyvecl v: pointer to input vector
+*              - const uint8_t seed: byte array containing seed
+*              - uint16_t nonce: 2-byte nonce
+**************************************************/
 void polyvecl_uniform_eta(ml_dsa_params *params,
                           polyvecl *v,
                           const uint8_t seed[CRHBYTES],
                           uint16_t nonce) {
   unsigned int i;
-
   for(i = 0; i < params->l; ++i)
     poly_uniform_eta(params, &v->vec[i], seed, nonce++);
 }
 
+/*************************************************
+* Name:        polyvecl_uniform_gamma1
+*
+* Description: FIPS 204: Algorithm 34 ExpandMask.
+*              Samples vector v with polynomial coordinates whose
+*              coefficients are in [-gamma1 + 1, gamma1].
+*
+* Arguments:   - ml_dsa_params: parameter struct
+*              - polyvecl v: pointer to input vector
```

Comment:
```suggestion
*              - polyvecl v: pointer to output vector
```

Reply:
fixed in [a8777cd](https://github.com/aws/aws-lc/pull/2017/commits/a8777cd91adbc6ddacbc0de027fce11eb8d3b4e7)